packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js

```diff
@@ -60,9 +66,7 @@ const CheckBoxView = ({
         [`${checkedSection}NameChecked`]: isChecked,
       })}
     >
-      {label ? (
-        <Typography className={classes.checkboxLabel}>{label}</Typography>
-      ) : (<Typography className={classes.checkboxName}>{name}</Typography>)}
+      <Typography className={classes.checkboxName}>{name}</Typography>
```

Comment:
@rana22, Looks like "label" is missing is that intentional or mistake? if intentional can you explain rational behind it?

Reply:
The label and name variables are redundant so only one is used.